CONTRIBUTING.md

```diff
@@ -368,8 +368,7 @@ For documentation strings, 🤗 Transformers follows the [google style](https://
 Check our [documentation writing guide](https://github.com/huggingface/transformers/tree/main/docs#writing-documentation---specification)
 for more information.
 
-#### This guide was heavily inspired by the awesome [scikit-learn guide to contributing](https://github.com/scikit-learn/scikit-learn/blob/main/CONTRIBUTING.md)
-
+<h4> This guide was heavily inspired by the awesome <a href=https://github.com/scikit-learn/scikit-learn/blob/main/CONTRIBUTING.md>scikit-learn guide to contributing</a></h4>
```

Comment:
This doesn't need to be a header (there is no reason to make it one). Let's just put this in bold and keep Markdown syntax.

Reply:
+1